libs/common/src/lib/retrieval-agent/agent-dashboard/workflow/basic-elements/node-form/node-form.component.scss

```diff
@@ -0,0 +1,17 @@
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
```

Comment:
We want to manage spacing consistently, so please do not set sizes manually.
You must import the variables:
```
@use 'apps/dashboard/src/variables' as *;
```

and then use the `rhythm` mixin:
```
gap: rhythm(2);
```
(see libs/sistema/theme/tokens/_spacing.tokens.scss for the sizes)